app/views/organizations/edit.html.erb

```diff
@@ -0,0 +1,55 @@
+<div class="row">
+  <div class="col-xs-12">
+    <%= render partial: "common/navbar", locals: { active: "organizations" } %>
```

Comment:
This needs to be removed to be compatible with #139 
I moved the navbar to the layout in that branch